server/test/support/tuist_test_support/fixtures/bundles_fixtures.ex

```diff
@@ -5,13 +5,25 @@ defmodule TuistTestSupport.Fixtures.BundlesFixtures do
   """
 
   alias Tuist.Bundles
+  alias TuistTestSupport.Fixtures.AccountsFixtures
   alias TuistTestSupport.Fixtures.ProjectsFixtures
 
   @doc """
   Generates a bundle.
   """
   def bundle_fixture(opts \\ []) do
     id = UUIDv7.generate()
+    project = Keyword.get(opts, :project, ProjectsFixtures.project_fixture())
+    
+    # Get or create an account for uploaded_by_account_id
+    uploaded_by_account = case Keyword.get(opts, :uploaded_by_account) do
+      nil -> 
+        case Keyword.get(opts, :uploaded_by_user) do
```

Comment:
why are we supporting `:uploaded_by_user`? the bundle table only has `uploaded_by_account_id`, so `:uploaded_by_account` is the only opt I'd keep for this.

I'd keep the onus of passing the account instead of the user on the tests and would keep the fixture more straightforward.